azure-pipelines.yml

```diff
@@ -151,8 +157,10 @@ jobs:
       $pyversion = python -c "from __future__ import print_function; import sys; print(sys.version.split()[0])"
       Write-Host "Python Version: $pyversion"
       $target = "C:\\hostedtoolcache\\windows\\Python\\$pyversion\\$(PYTHON_ARCH)\\lib\\openblas.a"
+      $target_dll = "C:\\hostedtoolcache\\windows\\Python\\$pyversion\\$(PYTHON_ARCH)\\lib\\openblas.dll"
       Write-Host "target path: $target"
       cp $tmpdir\$(BITS)\lib\libopenblas_v0.3.3-186-g701ea883-gcc_7_1_0.a $target
+      cp $tmpdir\$(BITS)\bin\libopenblas_v0.3.3-186-g701ea883-gcc_7_1_0.dll $target_dll
```

Comment:
this **really** needs to be clarified; while working on both NumPy / SciPy CIs I've noticed that we mostly just tend to move the static library file and build using that; so, I'm not clear on what exactly is going on with dynamic openblas builds & if we can viably use the DLL for version detection; just copying the DLL file in causes the Windows builds to break